### PR TITLE
Add missing docs

### PR DESCRIPTION
**Pull request by schustmi**
## Describe changes
I implemented/fixed _ to achieve _.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have based my new branch on `develop` and the open PR is targeting `develop`. If your branch wasn't based on develop read [Contribution guide on rebasing branch to develop](https://github.com/zenml-io/zenml/blob/main/CONTRIBUTING.md#-pull-requests-rebase-your-branch-on-develop).
- [ ] If my changes require changes to the dashboard, these changes are communicated/requested.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: docs/book/how-to/pipeline-development/trigger-pipelines/use-templates-cli.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+You can use the ZenML CLI to create a run template:
+
+```bash
+# The <PIPELINE_SOURCE_PATH> will be `run.my_pipeline` if you defined a
+# pipeline with name `my_pipeline` in a file called `run.py`
+zenml pipeline create-run-template <PIPELINE_SOURCE_PATH> --name=<TEMPLATE_NAME>
+```
+
+{% hint style="warning" %}
+You need to have a **remote stack** active while running this command or specify
```
Comment (bcdurak) [inline docs/book/how-to/pipeline-development/trigger-pipelines/use-templates-cli.md:21]:
```suggestion
You need to have an active **remote stack** while running this command or you can specify
```

File: src/zenml/cli/__init__.py [hunk 2]
```diff
@@ -2410,6 +2410,23 @@ def my_pipeline(...):
...
 
+If you want to create a run template for your pipeline which can then be used
+to trigger a run from the dashboard or via an HTTP request, use:
+
+```bash
+zenml pipeline create-run-template <PIPELINE_SOURCE_PATH> \
+   --name=<TEMPLATE_NAME>
+
+To specify a config file or create the template for a different stack than the
+active one, use the `--config/-c` and `--stack` options:
```
Comment (bcdurak) [inline src/zenml/cli/__init__.py:2421]:
```suggestion
To specify a config file, use the `--config/-c` option. If you would like to use a different stack than the active one, use the `--stack` option.
```

File: src/zenml/cli/__init__.py [hunk 3]
```diff
@@ -2410,6 +2410,23 @@ def my_pipeline(...):
 zenml pipeline run <PIPELINE_SOURCE_PATH> --stack=<STACK_ID_OR_NAME>
 ```
 
+If you want to create a run template for your pipeline which can then be used
+to trigger a run from the dashboard or via an HTTP request, use:
```
Comment (bcdurak) [inline src/zenml/cli/__init__.py:2414]:
```suggestion
If you want to create a run template based on your pipeline that can later be used to trigger a run either from the dashboard or through an HTTP request:
```

File: docs/book/how-to/pipeline-development/trigger-pipelines/README.md [hunk 4]
```diff
@@ -59,5 +59,6 @@ This is a [ZenML Pro](https://zenml.io/pro)-only feature. Please
 
 ![Working with Templates](../../../.gitbook/assets/run-templates.gif)
 
-<table data-view="cards"><thead><tr><th></th><th></th><th></th><th data-hidden data-card-target data-type="content-ref"></th></tr></thead><tbody><tr><td>Use templates: Python SDK</td><td></td><td></td><td><a href="use-templates-python.md">use-templates-python.md</a></td></tr><tr><td>Use templates: Dashboard</td><td></td><td></td><td><a href="use-templates-dashboard.md">use-templates-dashboard.md</a></td></tr><tr><td>Use templates: Rest API</td><td></td><td></td><td><a href="use-templates-rest-api.md">use-templates-rest-api.md</a></td></tr></tbody></table>
+<table data-view="cards"><thead><tr><th></th><th></th><th></th><th data-hidden data-card-target data-type="content-ref"></th></tr></thead><tbody><tr><td>Use templates: Python SDK</td><td></td><td></td><td><a href="use-templates-python.md">use-templates-python.md</a></td></tr><tr><td>Use templates: CLI</td><td></td><td></td><td><a href="use-templates-cli.md">use-templates-cli.md</a></td></tr>
```
[the remark this answers is not in the document]
Comment (schustmi) [reply docs/book/how-to/pipeline-development/trigger-pipelines/README.md]: Removed the newline

**Review by bcdurak — APPROVED** [review]

**Pull request merged**